src/main/java/seedu/duke/operations/Parser.java

```diff
@@ -0,0 +1,47 @@
+package seedu.duke.operations;
+
+import seedu.duke.command.*;
+
+/**
+ * Parser parses the user input and returns the relevant
+ * Command to execute.
+ */
+public class Parser {
+
+    /**
+     * Returns a Command after parsing user input.
+     *
+     * @param cmd   User input
+     * @return      Command
+     */
+    public static Command parse(String cmd) {
+        cmd = cmd.strip();
```

Comment:
Preferably do not use short forms.
```suggestion
    public static Command parse(String command) {
        command = command.strip();
```